### PR TITLE
[quorum] update quorum helm value templates for flux v2

### PR DESCRIPTION
**Pull request by manikanta-darsi**
Signed-off-by: manikanta-darsi <manikanta.darsi@accenture.com>

**Changelog**
- Update  quorum helm value templates for flux v2.

 

**Reviewed by**
@sownak @suvajit-sarkar 

 

**Linked issue**
#1928

### CODE REVIEW

**Review by suvajit-sarkar — APPROVED** [review]
LGTM

File: examples/supplychain-app/configuration/roles/helm_component/templates/frontend.tpl [hunk 1]
```diff
@@ -21,7 +25,7 @@ spec:
       nodePorts:
         port: {{ peer_frontend_port }}
         targetPort: {{ peer_frontend_targetport }}
-      image: {{ network.container_registry.url | lower }}/bevel-supplychain-frontend:latest
+      image: ghcr.io/hyperledger/bevel-supplychain-frontend:stable
```
Comment (suvajit-sarkar) [inline examples/supplychain-app/configuration/roles/helm_component/templates/frontend.tpl:28]: this should come from network yaml; and the ghcr repo should be added in samples

**Review by suvajit-sarkar — CHANGES_REQUESTED** [review]
See the inline comments

File: examples/supplychain-app/configuration/roles/helm_component/templates/frontend.tpl (hunk 1 above)
Comment (manikanta-darsi) [reply examples/supplychain-app/configuration/roles/helm_component/templates/frontend.tpl]: Changed.

**Review by suvajit-sarkar — APPROVED** [review]

File: platforms/quorum/configuration/reset-network.yaml [hunk 2]
```diff
@@ -26,8 +26,18 @@
         kubernetes: "{{ item.k8s }}"
       loop: "{{ network['organizations'] }}"
 
-    # Delete Helmreleases and uninstall flux
-    # Change this role if you have new Helmreleases to delete 
+    # Uninstalling Flux is needed so that everything is clean
```
Comment (jagpreetsinghsasan) [inline platforms/quorum/configuration/reset-network.yaml:29]: Can you update the same in the corresponding role Readme.md?

**Review by jagpreetsinghsasan — APPROVED** [review]

**Comment by jagpreetsinghsasan** [comment]
Just this one comment, rest everything looks good to me. Thanks for the PR

File: platforms/quorum/configuration/reset-network.yaml (hunk 2 above)
Comment (manikanta-darsi) [reply platforms/quorum/configuration/reset-network.yaml:29]: updated.

**Review by suvajit-sarkar — APPROVED** [review]
LGTM

**Pull request merged**